stdlib/domain.ml

```diff
@@ -129,6 +129,18 @@ module DLS = struct
 
 end
 
+(******** Identity **********)
+
+let get_id { domain; _ } = domain
+
+let self () = Raw.self ()
+
+external set_name : string -> unit = "caml_ml_domain_set_name"
```

Comment:
I suppose that this is not needed anymore that #11264 was merged?

Reply:
Removed.